docs/epidata_development.md

```diff
@@ -82,8 +82,12 @@ From the root of your workspace, all of the images can be built as follows:
 docker build -t delphi_web \
   -f repos/delphi/operations/dev/docker/web/Dockerfile .
 
-docker build -t delphi_web_epidata \
-  -f repos/delphi/delphi-epidata/dev/docker/web/epidata/Dockerfile .
+docker build -t delphi_web_python \
+  -f repos/delphi/delphi-epidata/dev/docker/python/Dockerfile .
+
+cd ./repos/delphi/delphi-epidata
+docker build -t delphi_web_epidata -f ./devops/Dockerfile .
+cd ../../../
```

Comment:
huh. does
```
docker build -t delphi_web_epidata \
   -f repos/delphi/delphi-epidata/devops/Dockerfile .
```

not work?

Reply:
It does work, but I decided to use the version in the CI Recipe [here](https://github.com/cmu-delphi/delphi-epidata/blob/76cc4c513fe1fa64eede08a6a9202aaa25e0dc1b/.github/workflows/ci.yaml#L58) but I'll update it to the more concise version